python/doc/pyplots/GaussianProcessFitter.py

```diff
@@ -0,0 +1,24 @@
+import openturns as ot
+from openturns.experimental import GaussianProcessFitter
+from openturns.viewer import View
+
+f = ot.SymbolicFunction(["x"], ["x * sin(x)"])
+sampleX = [[1.0], [2.0], [3.0], [4.0], [5.0], [6.0], [7.0], [8.0]]
+sampleY = f(sampleX)
+basis = ot.Basis(
+    [ot.SymbolicFunction(["x"], ["x"]), ot.SymbolicFunction(["x"], ["x^2"])]
+)
+covarianceModel = ot.SquaredExponential([1.0])
+covarianceModel.setActiveParameter([])
+algo = GaussianProcessFitter(sampleX, sampleY, covarianceModel, basis)
+algo.run()
+result = algo.getResult()
+
+graph = f.draw(0.0, 10.0)
+graph.add(result.getMetaModel().draw(0.0, 10.0))
+graph.add(ot.Cloud(sampleX, sampleY))
+graph.setColors(["blue", "red", "black"])
+graph.setLegends(["model", "meta model", "sample"])
```

Comment:
```suggestion
graph.setLegends(["model", "GP mean", "sample"])
```
Calling this "metamodel" is risky, people might think this is the final metamodel...